projects/ccf-eui/src/app/app.component.scss

```diff
@@ -14,7 +14,6 @@
   height: 100%;
   overflow: hidden;
   font-size: 1rem;
-  color: black;
```

Comment:
This was put here for the web components. What's the reason for removing this?

Reply:
The text in dark mode is showing as the wrong color. How do we get around this?

Reply:
In the EUI?

Reply:
Yes